docs/source/topic-guides/models.md

```diff
@@ -222,7 +222,7 @@ If there is only one _current owner_, the fulfillment will be a simple signature
 
 - `fid`: Fulfillment index. It matches a `cid` in the conditions with a new _crypto-condition_ that the new owner
   needs to fulfill to spend this asset.
-- `current_owners`: A list of public keys of the current owners; in this case it has just one public key.
+- `owners_before`: A list of public keys of the current owners; in this case it has just one public key.
```

Comment:
Maybe change this to:
 
`owners_before`: A list of public keys of the owners before the transaction; in this case it has just one public key.